data/Grigg_2008/metadata.yml

```diff
@@ -0,0 +1,547 @@
+    the surface of the interdunes, a lateritic gravel layer generally begins at 50
+    cm, eventually forming a conglomerated ironstone hardpan by 2-3 m. This hardpan
+    underlies the landscape as a flat pavement and continues beneath the dunes without
+    conforming to their shape. A white kaolinite/muscovite clay horizon is found typically
+    between 8 and 20 m below this hardpan layer and supports a perched water table
+    (Nifty Copper Mine Drill Log 2003). Average depth to water beneath most interdunes
+    throughout the area is ~10-12 m, and in some low-lying depressions maybe <5 m
+    from the surface (Nifty Copper Mine Drill Log 2003; Rockwater Pty Ltd 2003). Adjacent
+    dunes and interdunes were selected as study sites where flat sections across the
+    interdune exceeded 300 m and remained unburnt for at least 10 years. Vegetation
+    was very similar on all dunes throughout the area, with the most prominent species
+    being the eucalypt Corymbia chippendalei (Myrtaceae). These trees were only ever
+    found on the uppermost sections of the sandy ridges. Other common dune species
+    included Grevillea stenobotrya (Proteaceae), Acacia  jensenii (Fabaceae) and Triodia
+    schinzii (Poaceae). Most interdunes were characterised by scattered shrubs such
+    as Grevillea wickhamii (Proteaceae), Acacia ancistrocarpa (Fabaceae) and cover
+    was dominated by Triodia basedowii. In lower-lying areas of some interdunal flats,
+    small-statured Eucalyptus victrix and Melaleuca species (Myrtaceae) co-occurred
+    on gravely patches, with a brackish water table often within 5 m of the surface.
+    We carried out comparative measurements on plant water relations and nutrition
+    of three congeneric or closely related dune-interdune species pairs, representing
+    three different functional groups. During the course of 2 years, we examined water
+    relations and photosynthesis of six dominant woody species throughout each season,
+    in addition to foliar delta 13C, delta 15N and nutrient composition. Adjacent
+    dunes and interdunes were selected as study sites where flat sections across the
+    interdune exceeded 300 m and remained unburnt for at least 10 years. table often
+    within 5 m of the surface. Information on water and nutrient relations are presented
+    here for woody species of three functional types, including (1) the non-leguminous,
+    mycorrhizal, relatively large eucalypt trees (C. chippendalei and E. victrix),
+    (2) the smaller, non-mycorrhizal, cluster-bearing proteaceous tree-shrubs (G.
+    stenobotrya and G. wickhamii) and (3) the leguminous, mycorrhizal, multi-stemmed
+    shrubs (A. jensenii and A. ancistrocarpa). A companion paper presenting data for
+    the two spinifex grass species, T. schinzii and T. basedowii, is in preparation
+    (A. M. Grigg, unpubl. data)
+  original_file: none, data extracted from paper.
+  notes: data extracted from paper, not raw data
+locations:
```

Comment:
I decided to divide locations into the two communities, in order to map in all the soil information. I then remove dune vs interdune as a context